docs/policy/static-pools.md

```diff
@@ -3,18 +3,17 @@
 ## Overview
 
 The `static-pools` (STP) builtin policy was inspired by [CMK (CPU Manager for
-Kubernetes)][cmk]. It is an
-example policy demonstrating some capabilities of `cri-resource-manager` - not
-production ready.
+Kubernetes)][cmk]. It is an example policy demonstrating capabilities of
```

Comment:
...`][cmk]`
should probably be ...`](cmk)` in order to get a working link

Reply:
This works as it is because `cmk` is defined at the bottom of the file